### PR TITLE
No Organization Usage

### LINKED ISSUE #51: Support Init on `User` Organization
If the logged in user does not have any permission to create repos in any of the organizations, you get an error being thrown when inquirer prompts the organization question. Let's check if any orgs are present before prompting the questions and log a message. 

See image: https://searchspring.slack.com/archives/CRQDMSWBT/p1657634174867949

### PR DESCRIPTION
**Pull request by korgon**
Changes to `init` command:
* adding support for creating repositories in user github and organizations
* correcting HTTPS fallback when cloning (it will now ask for password)
* improved error logging

Other changes to color choices throughout.

### CODE REVIEW

File: src/cli.js
Comment (dkonieczek) [inline src/cli.js:47]: `recs sync` command will need to function without a logged in user (ie. when it runs in the action)

A simple solution would be to check if `--secrets-ci` is passed in

File: src/init.js
Comment (dkonieczek) [inline src/init.js:164]: `answers.organization` may show up at undefined here? Perhaps change to `${answers.organization || user.login}/${answers.name}`

**Review by dkonieczek — CHANGES_REQUESTED** [review]

File: src/cli.js
Comment (korgon) [reply src/cli.js:47]: Ah... yup. I'll have to wait until that change has been merged in I suppose.

File: src/init.js
Comment (korgon) [reply src/init.js:164]: This will always have something in it since the user login is now added as an option at the end of the list:
https://github.com/searchspring/snapfu/blob/5772c9d071264ca0c55e9bf53a0a53b34a5a0688/src/init.js#L112

**Review by dkonieczek — APPROVED** [review]

**Pull request merged**
